frontend/src/components/ProjectCreate.vue

```diff
@@ -58,6 +59,29 @@
           </div>
         </div>
       </div>
+      <div v-if="state.project.tenantMode === 'TENANT'" class="col-span-1">
+        <label
+          class="text-lg leading-6 font-medium text-control select-none flex items-center"
+        >
+          {{ $t("project.db-name-template") }}
+          <BBCheckbox
+            :value="state.enableDbNameTemplate"
+            class="ml-2"
+            @toggle="(on: boolean) => state.enableDbNameTemplate = on"
+          />
+        </label>
+        <p class="mt-1 text-sm font-normal">
+          {{ $t("project.create-modal.db-name-template-tips") }}
```

Comment:
Not in this PR, but I think after we add the content to doc.bytebase.com, we should add a link to the doc here